docs/language/api.rst

```diff
@@ -89,6 +89,15 @@ the error ``Keyword argument :foo needs a value``. To avoid this, you can quote
 the keyword, as in ``(f ':foo)``, or use it as the value of another keyword
 argument, as in ``(f :arg :foo)``.
 
+discard prefix
+--------------
+
+Hy supports the EDN discard prefix, like Clojure.
+Any form prefixed with ``#_`` is discarded instead of compiled.
+It's often more useful than linewise comments for commenting out a
+form, because it respects code structure even when part of another
+form is on the same line.
+
```

Comment:
Please include an example, and write out "Extensible Data Notation". I didn't know what "EDN" stands for, and I bet that most other readers won't, either.